privacyidea/lib/tokens/passwordtoken.py

```diff
@@ -49,6 +50,7 @@ class PasswordTokenClass(TokenClass):
     password_detail_key = "password"  # nosec B105 # key name
     default_length = DEFAULT_LENGTH
     default_contents = DEFAULT_CONTENTS
+    desc_key_gen = lazy_gettext("Force the key to be generated on the server.")
```

Comment:
We should probably define this as a module level variable in `hotptoken.py` and import it here or in the ApplSpec token module